docs/release-notes/index.md

```diff
@@ -21,6 +21,261 @@ To check for security updates, go to [Security announcements for the Elastic sta
+* Replaces registry call when deleting {{kib}} assets for custom packages [#224886]({{kib-pull}}224886).
+* Ensures package policy names are unique when moving across spaces [#224804]({{kib-pull}}224804).
+* Fixes bulk actions incorrectly selecting agents when a namespace filter is applied [#224036]({{kib-pull}}224036).
+* Reverts "Added instructions for installing {{agent}} complete." [#223520]({{kib-pull}}223520).
+* Fixes styled component theme lookup issue [#221979]({{kib-pull}}221979).
+* Fixes SSL config being overridden by advanced YAML in full agent policy [#219902]({{kib-pull}}219902).
+* Fixes required capability for the SIEM migrations topic [#219427]({{kib-pull}}219427).
+* Makes output and {{fleet}} Server settings non-editable for agentless policies [#218905]({{kib-pull}}218905).
+* Supports integrations with secrets that contain multiple values [#216918]({{kib-pull}}216918).
+* Adds remote cluster instructions for syncing integrations [#211997]({{kib-pull}}211997).
+* Updates install snippets to include all platforms [#210249]({{kib-pull}}210249).
+
+**Discover**:
+* Fixes invalid input highlight in **Data View** flyout [#226822]({{kib-pull}}226822).
+* Fixes an issue causing an internal server error when renaming a search session [#226757]({{kib-pull}}226757).
+* Fixes an issue causing **Discover** to freeze when dragging & dropping columns with animations disabled [#226592]({{kib-pull}}226592).
+* Fixes row highlighting when reordering columns [#226584]({{kib-pull}}226584).
+* Fixes an issue causing an error when updating then deleting a saved query [#226569]({{kib-pull}}226569).
+* Fixes a cell value alignment issue [#226562]({{kib-pull}}226562).
+* Adds missing information icon to the document viewer table [#222299]({{kib-pull}}222299).
+* Fixes an issue incorrectly showing the *unmapped* icon when a field changed from unmapped to mapped [#221308]({{kib-pull}}221308).
+* Fixes the parsing of index patterns in the **Inspect** feature of Kibana. Previously, certain index pattern strings were not being parsed and displayed correctly in the **Inspect** feature [#221084]({{kib-pull}}221084).
+* Fixes an issue that causes transitions from **Logs Stream** and **Logs Explorer** to **Discover** to lose some context such as the selected time range or KQL query [#215867]({{kib-pull}}215867).
+* Excludes only {{es}} metadata fields from the Summary column instead of all fields starting with `_` [#213255]({{kib-pull}}213255).
+* Fixes inability to clear Document ID in data view field editor preview [#220891]({{kib-pull}}220891).
+* Fixed multiple accessibility issues, including adding missing aria labels and column headers, improving keyboard navigation and interactions, and improving focus changes when interacting with **Discover** features [View list of fixes](https://github.com/elastic/kibana/issues?q=state:closed%20label:Project:Accessibility%20label:v9.1.0%20label:Team:DataDiscovery).
```

Comment:
@kertal @davismcphee this is the a11y fixes summary we had talked about over Slack, lmk how that sounds to you. I added the `Project:Accessibility` label to a few PRs that were missing it so I could link to the list of a11y fixes with a single link.

Reply:
Looks great! Thanks @florent-leborgne 🙏